src/main/java/com/fasterxml/jackson/core/base/ParserBase.java

```diff
@@ -800,6 +800,11 @@ public BigDecimal getDecimalValue() throws IOException
         return _getBigDecimal();
     }
 
+    @Override // @since 2.15
```

Comment:
Ok I merged this but then realized that maybe this is actually needed in base `JsonParser` itself?

Reply:
It looks like streamReadConstraints will need to move up JsonParser class. I'll do another PR.